cmd/addoperator.go

```diff
@@ -155,6 +155,9 @@ func (p *AddOperatorParams) Load(ctx ActionCtx) error {
 		}
 		op, err := jwt.DecodeOperatorClaims(token)
 		if err != nil {
+			if err.Error() == JWTV2DecodeError {
+				return fmt.Errorf("%sfailed to decode: %v", JWTUpgradeBannerJWT(), err)
```

Comment:
possibly - remove the new line from the format, since you are just adding a line, and add the \n here.